### DISCUSSION TITLE
Markdown or HTML for content

### DISCUSSION THREAD

**Discussion by shawnthompson** [gc-da11yn/gc-da11yn.github.io#196] [Development (Open discussions)]
I'd like to open up the discussion of switching to markdown for content of the site that doesn't need any special designs like columns.

@delisma and I were speaking and we think that it might be better to use markdown for all the content.

Once we get the use of it it's quicker to code with less room for error. 

I use [PuppyPaste](https://puppypaste.com/) to convert Word content to markdown. What do you use?

**Comment by delisma** [comment]
I use StackEdit to go from gDocs to Markdown. My workflow doesn't involve MS Office anymore. A lot of my Markdown editing is done straight into VS Code with a few Markdown plugins.

**Comment by delisma** [comment]
For more complex layouts, we could consider includes with variables. I'm starting to use this technique with Jekyll and so far it works well. I haven't tested the experience with a non developer as of yet, still exploring.

**Comment by shawnthompson** [comment]
I also think that maybe only main pages, which could be created using [collections](https://www.11ty.dev/docs/collections/) to create the content should use HTML and most of the other documents could just be using straight markdown.

**Comment by shawnthompson** [comment]
What would be cool is if we use something to convert Word Docs to markdown in the build.

**Reply by delisma** [reply]
I would be more incline to make people use a native Markdown editor instead or export from Word in plain text. Less dependencies.

**Reply by shawnthompson** [reply]
But if we can do it automatically because the content will be coming in Word and not markdown from the content group. Someone is going to have to do the conversion.

**Reply by delisma** [reply]
But why would you want to work in Word to start with? We could add the feature for sure or we can change the culture around the workflow. 

Build a car, not a faster horse.

**Reply by shawnthompson** [reply]
Baby steps sir, people (comms) are already iffy about Github asking them to use Markdown might be too much. 

The GoC is a Microsoft shop. Everything internally is either in a Word or PowerPoint document.

**Comment by shawnthompson** [comment]
On thing we noticed was that lists inside of lists can be tricky.

Example:

  1. Lorem ipsum dolor sit:
     * Lorem ipsum dolor sit.
     * Lorem ipsum dolor sit.
     * Lorem ipsum dolor sit.s.
  2. Lorem ipsum dolor sit:
     * LLorem ipsum dolor sit.
     * Lorem ipsum dolor sit.
     * Lorem ipsum dolor sit.

**Versus**

1. Lorem ipsum dolor sit:
  * Lorem ipsum dolor sit.
  * Lorem ipsum dolor sit.
  * Lorem ipsum dolor sit.s.
2. Lorem ipsum dolor sit:
  * LLorem ipsum dolor sit.
  * Lorem ipsum dolor sit.
  * Lorem ipsum dolor sit.

**Reply by delisma** [reply]
@shawnthompson So we published a [new experimental tool](https://doc-to-html.tbs.alpha.canada.ca/) to convert Docx to Canada.ca HTML. Check it out, because we could rig the tool to do Markdown as well.

**Reply by shawnthompson** [reply]
Didn't work. I got an error when I tried with the a11y Standard.

![image](https://user-images.githubusercontent.com/2575266/204360973-b4b1d8cf-a761-46f9-8cb1-31179757ba66.png)


Alt text:

**Internal Server Error**
The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.

**Reply by delisma** [reply]
I think you have to use the Word template provided first after and then port your content into it.

**Comment by shawnthompson** [comment]
We've decided on using Markdown for as much as possible on the site
